lib/Migration/Version021200Date20240522000000.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\Collectives\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version021200Date20240522000000 extends SimpleMigrationStep {
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		if (!$schema->hasTable('collectives_sessions')) {
+			$table = $schema->createTable('collectives_sessions');
+			$table->addColumn('id', Types::BIGINT, [
+				'autoincrement' => true,
+				'notnull' => true,
+			]);
+			$table->addColumn('user_id', Types::STRING, [
+				'notnull' => true,
+				'length' => 64,
+			]);
+			$table->addColumn('token', Types::STRING, [
+				'notnull' => true,
+				'length' => 64,
```

Comment:
32?

Reply:
Not sure. The deck app uses a column size of `64`, so I copied that. Since we generate a random string with 32 characters, DB length of 32 should be enough in theory. I don't know enough about the internals if that's really the case though and feel a bit insecure to limit the DB field size that much. If you think it's safe, I'll change it :grimacing:

Reply:
Since regular ascii chars are being used, that should be fine, but if ever the token length would be increased or multi-byte characters added, it would require a migration change. I would go with 32, but I do not have a strong opinion either. 

In DirectEditing, for example, length is 64, but the token is also generated with 64 characters (but the chosen pool for the token has 4 characters less)